scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -20,6 +20,7 @@ workflow GvsExtractCallset {
     Int? scatter_count
     Int? extract_memory_override_gib
     Int? disk_override
+    Boolean bgzip_output_vcfs = true
```

Comment:
The ticket says this should be off by default, although if we're running this WDL singly we're *probably* doing an AoU "small callset" where we actually do want this on...

So I'm not sure if we should leave this as you have it, or default to `false` and update the (currently nonexistent) AoU small callset docs to specify that this should be set to `true`?

Reply:
Thanks. That's what I was thinking. That is, have it off by default, but enable it here for the likely AoU small call sets case. I agree that it should be off by default.